Rakefile

```diff
@@ -11,7 +11,8 @@ Vmdb::Application.load_tasks
 begin
   require 'jasmine'
   load 'jasmine/tasks/jasmine.rake'
-rescue LoadError
+rescue LoadError => e
+  puts e.message
```

Comment:
Yes, this is so that we can run jasmine in environments that support it, and complaining every time about not being able to load it everywhere else is *not* the right thing to do.

So.. `# Do nothing` is fine :)

I see no need to *fix all the rubocops*, so I'd prefer a simple `Do nothing` to a less readable `rubocop:disable ArbitraryRuleName`, but, well, either works.